types/vec2/index.d.ts

```diff
@@ -0,0 +1,229 @@
+// Type definitions for vec2 1.6
+// Project: https://github.com/tmpvar/vec2.js
+// Definitions by: Konrad Klockgether <https://github.com/Nielio>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/**
+ * A generic library useful when you need to work with points/vectors in 2d space.
+ * **Stuff to Note**: most of the Vec2's methods take a `returnNew` as the last parameter.  If passed a truthy value, a new vector will be returned to you.
+ * Otherwise the operation will be applied to `this` and `this` will be returned.
+ * Also, since `Infinity`and `NaN` are so insidious, this library will throw as soon as it detects either of these so you can take action to fix your data/algorithm.
+ */
+declare class Vec2 {
+    x: number;
```

Comment:
`public readonly`?

Reply:
The library allows to write directly into theese attributes.
But since, there is a function `change()` witch sets the values, i decided to hint them as `readonly`, though.
Also i added the `public` keyword everywhere.